schemas/2015-07-01-preview/Microsoft.ServerManagement.json

```diff
@@ -67,6 +67,45 @@
         "location",
         "properties"
       ]
+    },
+    "session": {
```

Comment:
you need to add reference to this resource type into 2015-01-01\deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate.json so that people can see the new type in VS.